src/main/java/net/mcreator/ui/modgui/MusicDiscGUI.java

```diff
@@ -125,31 +126,31 @@ public MusicDiscGUI(MCreator mcreator, ModElement modElement, boolean editingMod
 		ComponentUtils.deriveFont(name, 16);
 
 		subpane2.add(HelpUtils.wrapWithHelpButton(this.withEntry("musicdisc/sound"),
-				new JLabel("<html>Sound to play:<br><small>Music needs to be mono to properly work")));
+				new JLabel(L10N.t("elementgui.music_disc.music_mono_tip"))));
 		subpane2.add(music);
 
-		subpane2.add(HelpUtils.wrapWithHelpButton(this.withEntry("common/gui_name"), new JLabel("Name in GUI:")));
+		subpane2.add(HelpUtils.wrapWithHelpButton(this.withEntry("common/gui_name"), new JLabel(L10N.t("elementgui.music_disc.name_in_gui"))));

```

Comment:
`Name in GUI:` is used a lot, I think we should use a common key for it

Reply:
Okay I'm gonna setup a common key for it

Reply:
Done!